src/oscar/apps/catalogue/migrations/0025_attribute_code_uniquetogether_constraint.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 3.2.9 on 2022-01-25 20:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('catalogue', '0024_remove_duplicate_attributes'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='attributeoption',
+            name='id',
+            field=models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID'),
+        ),
+        migrations.AlterField(
+            model_name='attributeoptiongroup',
+            name='id',
+            field=models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID'),
```

Comment:
@joeyjurjens please remove these alterations that you did not explicitly define. We don't want them right now, you can control the autofield with a setting in django